pydantic_ai_slim/pydantic_ai/toolsets/function.py

```diff
@@ -77,7 +81,7 @@ def tool(
         retries: int | None = None,
         prepare: ToolPrepareFunc[AgentDepsT] | None = None,
         docstring_format: DocstringFormat = 'auto',
-        require_parameter_descriptions: bool = False,
+        require_parameter_descriptions: bool | None = None,
         schema_generator: type[GenerateJsonSchema] = GenerateToolJsonSchema,
```

Comment:
I think it'd make sense to also add toolset-wide defaults for `docstring_format` and `schema_generator`